examples/step-12/doc/intro.dox

```diff
@@ -54,7 +54,7 @@ On each cell $T$, we multiply by a test function $v_h$ from the left and integra
 to get:
 @f[
   \left( v_h, \nabla \cdot (\beta u_h) \right)_T
-= -(\nabla v_h, \beta u_h) + \int_\Gamma v_h u_h \beta \cdot n
+= -(\nabla v_h, \beta u_h) + \int_{\partial T} v_h u_h \beta \cdot n
```

Comment:
The tutorial has `\Gamma  = \partial T` a few lines above.

Reply:
That is not true. `\Gamma = \partial\Omega` is defined.

Reply:
Fair enough.